serverless.yaml

```diff
@@ -152,8 +167,10 @@ resources:
               KeyType: HASH
             - AttributeName: vid
               KeyType: RANGE
-          BillingMode: PAY_PER_REQUEST
           TableName: '${self:custom.resourceTableName}'
+          ProvisionedThroughput:
+            ReadCapacityUnits: 10
+            WriteCapacityUnits: 10

```

Comment:
I'd rather avoid this change unless it's absolutely required. Have you tested glue ETL with on-demand? From the AWS docs it looks like it should work

> When the DynamoDB table is in on-demand mode, AWS Glue handles the read capacity of the table as 40000. 

For exporting a large table we recommend switching your DynamoDB table to on-demand mode.
https://docs.aws.amazon.com/glue/latest/dg/aws-glue-programming-etl-connect.html#aws-glue-programming-etl-connect-dynamodb

Reply:
Ohh good find! In the tutorials that I've been following they said that DDB tables had to be in `ProvisionedThroughput` mode. But it looks like since the the tutorials were written, Glue now supports on-demand DDB tables. I just tested Glue with an on-demand DDB table, and it works.